pages/examples/withplatform/geoProcessGraphic.html

```diff
@@ -0,0 +1,119 @@
+            <div id="geo-process-graphic" style="margin: 80px; width: calc(100% - 160px); height: calc(100% - 160px);"></div>
+        </main>
+        <script type="text/javascript">
+          window.onload = () => {
+            initAuth('Geo-Process Graphic');
+            let tsiClient = new TsiClient();
+            let environmentFqdn = 'd7603093-b261-4f9e-b1f2-2596dfd88289.env.timeseries.azure.com';
+            let container = document.getElementById('geo-process-graphic');
+            
+            let geoProcessGraphic = tsiClient.ux.GeoProcessGraphic(container);
+            let geoProcessGraphicOptions = {
+                subscriptionKey: 'JEvUCDxRV-vlQI4cyPQqH_WVsupUriYWOicuQceCsew',
+                zoom: 3,
+                view: 'auto',
+                width: 1900,
+                height: 800,
+                theme: 'light',
+                throttleSlider: true,
+                xAxisHidden: false,
+                bucketSizeMillis: 21600000,
+                updateInterval: 1500,
+            };
+            let tsqExpression1 = new tsiClient.ux.TsqExpression(
+              {timeSeriesId: ['9461776'] },
+              {
+                LatitudeHoldValue: {
+                    kind: 'numeric',
+                    value: { tsx: '$event.lat.Double' },
+                    aggregation: {
+                      "tsx": "right($value)"
+                    },
+                    interpolation: {
+                        "kind": "linear",
+                        "boundary": {
+                          "span": "P30D"
+                        }
+                    }
+                },
+                LongitudeHoldValue: {
+                    kind: 'numeric',
+                    value: { tsx: '$event.lon.Double' },
+                    aggregation: {
+                      "tsx": "right($value)"
+                    },
+                    interpolation: {
+                        "kind": "linear",
+                        "boundary": {
+                          "span": "P30D"
+                        }
+                    }
+                },
+                AirTempHoldValue: {
+                  kind: 'numeric',
+                  value: { tsx: "$event['amb_air_temp'].Double" },
+                  aggregation: {
+                    "tsx": "right($value)"
+                  },
+                  interpolation: {
+                      "kind": "linear",
+                      "boundary": {
+                        "span": "P30D"
+                      }
+                  }
+                }
+              },
+              { },
+              {
+                positionXVariableName : 'LatitudeHoldValue',
+                positionYVariableName: 'LongitudeHoldValue',
+                alias: 'Titanic'
+              }
+            );
+
+            let image =  '/images/boat.png';
+            geoProcessGraphic.render(environmentFqdn, authContext.getTsiToken, image, [tsqExpression1], geoProcessGraphicOptions);
```

Comment:
this should be a chartDataOption, not a parameter, so it should probably be below "alias" and have an appropriate name like "image": '/images/boat.png'